src/rtr/server.rs

```diff
@@ -334,10 +341,10 @@ where Sock: AsyncRead + Unpin {
                 Ok(())
             }
         }
-        else if header.version() > 2 {
+        else if header.version() > MAX_VERSION {
             Err(Query::Error(
                 pdu::Error::new(
-                    header.version(),
+                    MAX_VERSION, // respond with what they should try
```

Comment:
`MAX_VERSION` should also be used in the error string 3 lines below.

Reply:
Good point. Fixed.